README.md

```diff
@@ -1,3 +1,5 @@
+[![Go Report Card](https://goreportcard.com/badge/github.com/storageos/operator)](https://goreportcard.com/report/github.com/storageos/operator) [![Lint and test](https://github.com/storageos/operator/actions/workflows/test.yml/badge.svg)](https://github.com/storageos/operator/actions/workflows/test.yml) [![CodeQL](https://github.com/storageos/operator/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/storageos/operator/actions/workflows/codeql-analysis.yml) [![Active](http://img.shields.io/badge/Status-Active-green.svg)](https://github.com/storageos/operator) [![PR's Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/storageos/operator/pulls) [![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
```

Comment:
Regarding the `Lint and test` badge, I'm not sure if we should also have the e2e tests action badge or not have any badge for these different types of test actions. We may have more actions for testing more different things in the future, like OLM installation test.

Reply:
I see. but as i know actions doesn't have global badge, only per action. So what is your suggestion?

Reply:
Maybe include both normal test and e2e test badges for now.